### PR TITLE
Randy README.md

### PR DESCRIPTION
**Pull request by RandyCastr0**
[Randy README 2.md](https://github.com/code-differently/code-differently-24-q1/files/14425190/Randy.README.2.md)

### CODE REVIEW

**Comment by RandyCastr0** [comment]
fixed broken image

File: lesson_00/RandyCastro/Randy Readme.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+## My work hours: 🗓️
+
+I prefer to kickstart my day with a productive morning routine. Typically, I hit the gym around 7 am to get energized. Then, from 9 am to 5 pm, I focus on tackling my tasks and projects with full dedication and efficiency.
+## One of my goals for this year🏆
+My goal is to successfully complete this program and secure a fulfilling career job in the tech industry. I aim to leverage this opportunity and eventually explore other avenues for personal and professional growth 
+
+## Family and Pets
+ In my household it is just my wife and our cat name Nori
+
+![Image Description](https://github.com/RandyCastr0/pics/raw/main/IMG_0786.jpg)
```
Comment (anthonydmays) [inline lesson_00/RandyCastro/Randy Readme.md:25]: This image is brokken. You should probably use a repo relative link here.

**Review by anthonydmays — CHANGES_REQUESTED** [review]

**Comment by RandyCastr0** [comment]
[Randy README 3.md](https://github.com/code-differently/code-differently-24-q1/files/14425987/Randy.README.3.md)

File: lesson_00/RandyCastro/Randy Readme.md
Comment (anthonydmays) [inline lesson_00/RandyCastro/Randy Readme.md:1]: The name of this file should be README.md. Please rename

**Review by anthonydmays — CHANGES_REQUESTED** [review]

**Comment by RandyCastr0** [comment]
[ README .md](https://github.com/code-differently/code-differently-24-q1/files/14426084/README.md)

**Review by anthonydmays — APPROVED** [review]
LGTM

**Review by RandyCastr0 — COMMENTED** [review]
Changed file name and update code

**Pull request merged**